api/osrd_infra/migrations/0015_railjson_v3_1.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.1.4 on 2022-12-12 16:44
+
+from django.db import migrations, models
+
+
+def apply_migration(apps, schema_editor):
+    """Update infra from v3.0.1 to v3.1.0"""
+    Infra = apps.get_model("osrd_infra", "Infra")
+    RouteModel = apps.get_model("osrd_infra", "RouteModel")
+    updated_infra = []
+
+    for infra in Infra.objects.all():
+        if infra.railjson_version == "3.0.1":
+            infra.railjson_version = "3.1.0"
+            updated_infra.append(infra)
```

Comment:
does it make sense to bump the version number of existing infras when the route format was not updated?
In my mind, keeping the version number as is has the benefit of letting us know which infras are supposed to work, and which need migration.

Reply:
It's indeed a better idea. We need to do few things before:

- Check in editoast that the modified infra has the right railjson version (otherwise we will have some errors)
- Have something in the front that prevent the user to use these infra

Reply:
I would do that in another PR.